src/update_handler.c

```diff
@@ -323,6 +323,7 @@ static gboolean casync_extract_image(RaucImage *image, gchar *dest, int out_fd,
 		 * file systems, additional mounts, etc. */
 		if (!seedslot->mount_point) {
 			g_debug("Mounting %s to use as seed", seedslot->device);
+			seedslot->extra_mount_opts = r_get_slot_current_mount_options(seedslot);
```

Comment:
We can't just replace `extra_mount_opts` here. Also, we should just use read-only mounts for seeding.

Reply:
It is now extended instead of being replaced.

As it accepted read-write slots before and we cannot easily determine what is actually readwrite and what is not (e.g. readonly file systems) I have left this untouched for now.

Reply:
What I meant is that `extra_mount_opts` is configuration, so we can't change it here. Also we don't need to check if the slot is currently mounted read-only, as an additional read-only mount should always work.